### PR TITLE
[release/10.6.8] Bug AB#95507: [Journey][ME&Back-Office] - Quiz> Scan-ID Symbol different between Back-office and ME

### PR DESCRIPTION
**Pull request by raoufr**
add scan-id icon

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,14 @@
 # Change Log
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## 10.6.8
+_June 22, 2023_
+- Add new icon type `scan-id`.  [PR #432](https://github.com/saddlebackdev/react-cm-ui/pull/441)
```
Comment (groberts314) [inline CHANGELOG.md:6]: We also have to add an entry about my bug fix too (from #440).
I usually do this separately.  Like I start release branch, bump the version number in `package.json` and `versions.js` (`npm install` so `package-lock.json` also has no version, and do this in a commit in release branch, then finish the release and publish the package.
See https://github.com/saddlebackdev/react-cm-ui/wiki/Deployment-And-Publishing#step-by-step

**Review by groberts314 — APPROVED** [review]
I'm approving, but please ensure we update change log about my bug fix in #440 as well.

**Comment by raoufr** [comment]
> I'm approving, but please ensure we update change log about my bug fix in #440 as well.

Will do @groberts314, no worries.
thank you

File: CHANGELOG.md (hunk 1 above)
Comment (raoufr) [reply CHANGELOG.md:6]: Add a new entry for PR #440 in the CHANGELOG

Comment (groberts314) [reply CHANGELOG.md]: Looks good.  Resolving.

**Comment by groberts314** [comment]
We also need to:
* `npm install` and commit `package-lock.json` with updated version number
* Update `/src/versions.js`:
https://github.com/saddlebackdev/react-cm-ui/blob/158ea39ebfa04762c6ebf3d9135d57a474d26177/src/versions.js#L19-L21

**Comment by raoufr** [comment]
> We also need to:
> 
> * `npm install` and commit `package-lock.json` with updated version number
> * Update `/src/versions.js`:
>   https://github.com/saddlebackdev/react-cm-ui/blob/158ea39ebfa04762c6ebf3d9135d57a474d26177/src/versions.js#L19-L21
> 
> Next time please let's stick to the documented [Deployment Procedures](https://github.com/saddlebackdev/react-cm-ui/wiki/Deployment-And-Publishing#step-by-step) and keep the version and change log changes separate from feature changes.

Sounds good

**Comment by raoufr** [comment]
> We also need to:
> 
> * `npm install` and commit `package-lock.json` with updated version number
> * Update `/src/versions.js`:
>   https://github.com/saddlebackdev/react-cm-ui/blob/158ea39ebfa04762c6ebf3d9135d57a474d26177/src/versions.js#L19-L21
> 
> Next time please let's stick to the documented [Deployment Procedures](https://github.com/saddlebackdev/react-cm-ui/wiki/Deployment-And-Publishing#step-by-step) and keep the version and change log changes separate from feature changes.

@groberts314 the version.js and package-lock.json are updated now.

**Review by groberts314 — APPROVED** [review]

**Pull request merged**